spec/API_specification/signatures/extension_linear_algebra_functions.py

```diff
@@ -0,0 +1,498 @@
+
+    .. note::
+       Whether an array library explicitly checks whether an input array is a symmetric matrix (or a stack of symmetric matrices) is implementation-defined.
+
+    Parameters
+    ----------
+    x: array
+        input array having shape ``(..., M, M)`` and whose innermost two dimensions form square matrices. Must have a floating-point data type.
+
+    Returns
+    -------
+    out: Tuple[array]
+        a namedtuple (``eigenvalues``, ``eigenvectors``) whose
+
+        -   first element must have the field name ``eigenvalues`` (corresponding to ``L`` above) and must be an array consisting of computed eigenvalues. The array containing the eigenvalues must have shape ``(..., M)``.
+        -   second element have have the field name ``eigenvectors`` (corresponding to ``Q`` above) and must be an array where the columns of the inner most matrices contain the computed eigenvectors. These matrices must be orthogonal. The array containing the eigenvectors must have shape ``(..., M, M)``.
+
+        Each returned array must have the same floating-point data type as ``x``.
+
+    .. note::
+       Eigenvalue sort order is left unspecified and is thus implementation-dependent.
+    """
+
+def eigvalsh(x: array, /) -> array:
+    """
+    Returns the eigenvalues of a symmetric matrix (or a stack of symmetric matrices) ``x``.
+
+    .. note::
+       The function ``eigvals`` will be added in a future version of the specification, as it requires complex number support.
+
+    ..
+      NOTE: once complex numbers are supported, each square matrix must be Hermitian.
+
+    .. note::
+       Whether an array library explicitly checks whether an input array is a symmetric matrix (or a stack of symmetric matrices) is implementation-defined.
+
+    Parameters
+    ----------
+    x: array
+        input array having shape ``(..., M, M)`` and whose innermost two dimensions form square matrices. Must have a floating-point data type.
+
+    Returns
+    -------
+    out: array
+        an array containing the computed eigenvalues. The returned array must have shape ``(..., M)`` and have the same data type as ``x``.
+
+    .. note::
+       Eigenvalue sort order is left unspecified and is thus implementation-dependent.
+    """
+
+def inv(x: array, /) -> array:
+    """
+    Returns the multiplicative inverse of a square matrix (or a stack of square matrices) ``x``.
+
+    Parameters
+    ----------
+    x: array
+        input array having shape ``(..., M, M)`` and whose innermost two dimensions form square matrices. Should have a floating-point data type.
+
+    Returns
+    -------
+    out: array
+        an array containing the multiplicative inverses. The returned array must have a floating-point data type determined by :ref:`type-promotion` and must have the same shape as ``x``.
+    """
+
+def matmul(x1: array, x2: array, /) -> array:
+    """
+    Alias for :func:`signatures.linear_algebra_functions.matmul`.
```

Comment:
This isn't creating a clickable link for me.

Also we should use `~signatures.linear_algebra_functions.matmul`. The `~` make it only show `matmul` in the text.